ui/web/src/services/RestServices/index.js

```diff
@@ -3,17 +3,17 @@ import axios from 'axios';
 export default class RestServices{
     constructor(){
         this.http = axios.create({
-            baseUrl: process.env.VUE_APP_BASE_URL,
-            headers:{ Authorization: `Bearer ${proccess.env.VUE_APP_ACCESS_TOKEN}` }
+            baseURL: `${process.env.VUE_APP_BASE_URL}`,
```

Comment:
Not necessary to use template literal string in baseURL

Reply:
Fixed